### ISSUE TITLE
Running `mypy --strict`, `list` with no type arguments gets the wrong error message with `parameters` instead of `arguments`

### ISSUE THREAD

**Issue by hyperkai** [python/mypy#20493]
*Memo:
- mypy --strict test.py
- mypy 1.19.1
- Python 3.14
- Windows 11

Running `mypy --strict`, `list` with no type arguments gets the wrong error message with `parameters` instead of `arguments` as shown below:

```python
v: list = [0, 1, 2]
```

> error: Missing type parameters for generic type "list"

So, the error message should be with `arguments` instead of `parameters` as shown below:

> error: Missing type arguments for generic type "list"

**Comment by A5rocks** [comment]
Those are called parameters.
